packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -244,6 +244,15 @@ function withAffectedOptions(yargs: yargs.Argv): yargs.Argv {
       coerce: parseCSV,
       default: []
     })
+    .options('runner', {
+      describe: 'This is the name of the tasks runner configured in nx.json',
+      type: 'string'
+    })
+    .options('configuration', {
```

Comment:
just so I understand. We are able to pass the configuration right now, we just don't use it for filtering.

Reply:
Yup